docs/facts/faq.rst

```diff
@@ -19,24 +19,36 @@ How to speed up a bit the process/test?
 I have multiple CPU
 ^^^^^^^^^^^^^^^^^^^
 
-Simply use the :code:`--multiprocess` argument to activate the usage of multiple processes.
-You can in addition of that use the :code:`--processes` argument to tell us the number of simultaneous process to run.
+Simply use the :code:`-m | --multiprocess` argument to activate the usage of multiple processes.
+You can in addition to that use the :code:`-p | --processes` argument to tell us the number of simultaneous process to run.
```

Comment:
s/argument to tell us the number of simultaneous process/argument to set the number of simultaneous process/